xlive/Blam/Engine/game/cheats.h

```diff
@@ -1,37 +1,41 @@
 #pragma once
 
-#include "Blam/Engine/game/game_globals.h"
+#define SKULL_COUNT 16
```

Comment:
noticed you want to follow the bungie naming scheme, thus this should be `k_skull_count`

Reply:
renamed